log-factory/src/main/java/io/deephaven/internal/log/Bootstrap.java

```diff
@@ -3,15 +3,19 @@
  */
 package io.deephaven.internal.log;
 
-class Bootstrap {
-    private static boolean isEnabled() {
-        return Boolean.parseBoolean(
-                System.getProperty("io.deephaven.internal.log.Bootstrap.enabled", "true"));
+public final class Bootstrap {
+
+    public static boolean isQuiet() {
+        return Boolean.getBoolean("deephaven.quiet");
     }
 
     public static void log(Class<?> source, String message) {
-        if (isEnabled()) {
-            System.out.printf("# %s: %s%n", source.getName(), message);
+        printf("# %s: %s%n", source.getName(), message);
+    }
+
+    public static void printf(String format, Object ... args) {
+        if (!isQuiet()) {
+            System.out.printf(format, args);
```

Comment:
Should this move to `System.err`?

Reply:
I think yes, that's the better long-term plan.

Bender had reasons to want to change it to stdout for web ui, and I think we'll want to fix those UX concerns first. https://github.com/deephaven/deephaven-core/pull/2743

Reply:
Darn, we probably shouldn't have merged that change.

Reply:
#3055